src/Sylius/Bundle/CoreBundle/Fixture/OrderFixture.php

```diff
@@ -111,42 +110,24 @@ public function __construct(
         $this->orderShippingMethodSelectionRequirementChecker = $orderShippingMethodSelectionRequirementChecker;
         $this->orderPaymentMethodSelectionRequirementChecker = $orderPaymentMethodSelectionRequirementChecker;
```

Comment:
I would vote for removal of all these fields and leave only `$this->orderExampleFactory` as a class field.